README.md

```diff
@@ -457,6 +472,149 @@ Uploads interrupted either due to bad internet connection or manual interruption
+
+`gsync folder_name -d gdrive_folder`
+
+Here, folder_name is the local folder you want to sync and gdrive_folder is google drive folder name.
+
+In the local folder, all the contents present or added in the future will be automatically uploaded.
+
+Note: Giving gdrive_folder is optional, if you don't specify a name with -d/--directory flags, then it will take the name of the local folder.
+
+Also, gdrive folder creation works in the same way as gupload command.
+
+Default wait time: 3 secs ( amount of time to wait before checking new files ).
+
+Default gupload arguments: None ( see -a/--arguments section below ).
+
+<a id="custom-flags-sync"></a>
+#### Custom Flags
+
+Read this section thoroughly to fully utilise the sync script, feel free to open an issue if any doubts regarding the usage.
+
+<details>
+
+<summary>Click to expand</summary>
+
+-   <strong>-d | --directory</strong>
+
+    Specify gdrive folder name, if not specified then local folder name is used.
+
+    ---
+
+-   <strong>-j | --jobs</strong>
+
+    See all background jobs that were started and still running.
+
+    Use -j/--jobs v/verbose to show additional information for jobs.
+
+    Additional information includes: CPU usage & Memory usage and No. of failed & successful uploads.
+
+    ---
+
+-   <strong>-p | --pid</strong>
+
+    Specify a pid number, used for --jobs or --kill or --info flags, multiple usage allowed.
+
+    ---
+
+-   <strong>-i | --info</strong>
+
+    Print information for a specific job. These are the methods to do it:
+
+    -   By specifying local folder and gdrive folder of an existing job,
+
+        e.g: `gsync local_folder -d gdrive folder -i`
+
+    -   By specifying pid number,
+
+        e.g: `gsync -i -p pid_number`
+
+    -   To show info of multiple jobs, use this flag multiple times,
+
+        e.g: `gsync -i pid1 -p pid2 -p pid3`. You can also use it with multiple inputs by adding this flag.
+
+    ---
+
+-   <strong>-k | --kill</strong>
+
+    Kill background jobs, following are methods to do it:
+
+    -   By specifying local_folder and gdrive_folder,
+
+        e.g. `gsync local_folder -d gdrive_folder -k`, will kill that specific job.
+
+    -   pid ( process id ) number can be used as an additional argument to kill a that specific job,
+
+        e.g: `gsync -k -p pid_number`.
+
+    -   To kill multiple jobs, use this flag multiple times,
+
+        e.g: `gsync -k pid1 -p pid2 -p pid3`. You can also using it with multiple inputs with this flag.
+
+    -   This flag can also be used to kill all the jobs,
+
+        e.g: `gsync -k all`. This will stop all the background jobs running.
+
+    ---
+
+-  <strong>-t | --time time_in_seconds</strong>
+
+    The amount of time that sync will wait before checking new files in the local folder given to sync job.
+
+    e.g: `gsync -t 4 local_folder`, here 4 is the wait time.
+
+    To set default time, use `gsync local_folder -t default=4`, it will stored in your default config.
+
+    ---
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[rule-style] Rules should use `---`](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5619177)